services/director-v2/tests/integration/02/utils.py

```diff
@@ -106,7 +107,7 @@ async def _try_to_clean():
     # in the on fixture teardown, relaxing a bit
     # this is mainly used for keeping the
     # dev environment clean
-    with suppress(aiodocker.DockerError):
+    with suppress(aiodocker.DockerError, RetryError):
```

Comment:
@GitHK I think the problem is that retry has the option`reraise=False,` and for that reason it does not raise the dockereror

Reply:
Thanks, let's see if it's better now